src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -205,6 +209,13 @@ def __init__(self, algorithm_name, optimize_mode='minimize'):
         self.rval = None
         self.supplement_data_num = 0
 
+        self.parallel = parallel_optimize
+        if self.parallel:
+            self.CL_rval = None
+            self.constant_liar_type = constant_liar_type
+            self.running_data = []
+            self.optimal_y = 0
```

Comment:
suggest to init self.optimal_y as `None`, because it is possible that the reported final result is 0. @PurityFan

Reply:
I think there is a potential problem if self.optimal_y is 0, because you cannot make sure the reported final result must be larger than 0.

Reply:
already changed.